comp/otelcol/otlp/collector.go

```diff
@@ -70,7 +70,7 @@ func (t *tagEnricher) Enrich(_ context.Context, extraTags []string, dimensions *
 	enrichedTags = append(enrichedTags, extraTags...)
 	enrichedTags = append(enrichedTags, dimensions.Tags()...)
 
-	entityTags, err := tagger.Tag(dimensions.OriginID(), t.cardinality)
+	entityTags, err := tagger.LegacyTag(dimensions.OriginID(), t.cardinality)
```

Comment:
Shouldn't we try to convert to entityID here to avoid calling `LegacyTag` from Go code?

Reply:
Right, updated